lib/clearLocalCache/index.js

```diff
@@ -0,0 +1,5 @@
+export default function clearLocalCache() {
```

Comment:
Since this is a cleanup method we just show a warning if it gets called in a web environment.